src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/WorkflowHistoryController.scala

```diff
@@ -33,6 +33,11 @@ class WorkflowHistoryController @Inject()(workflowHistoryService: WorkflowHistor
     workflowHistoryService.getHistoryForWorkflow(workflowId).toJava.toCompletableFuture
   }
 
+  @GetMapping(path = Array("/historyWorkflow"))
```

Comment:
I think it should be `/workflowFromHistory`, similar to the endpoint below. `/historyWorkflow` is confusing, especially because there is also `/workflowHistory`. Another possibility is `/workflowHistoryEntry`

Reply:
Fixed